lib/drivers/tandem/tandemDriver.js

```diff
@@ -599,6 +599,18 @@ module.exports = function (config) {
       name: 'CGM Calibration',
       format: 'iiihh',
       fields: ['current_time','timestamp','cal_timestamp','cal_reading', 'value']
+    },
+    LID_CGM_DATA_GX: {
+      value: 0xD3,
+      name: 'CGM GX Data Sample',
+      format: 'sby.ysii',
+      fields: ['status','type','rate','rssi','value','timestamp', 'transmitter_timestamp']
+    },
+    LID_CGM_CALIBRATION_GX: {
+      value: 0xD2,
+      name: 'CGM Calibration GX',
+      format: 's..............',
+      fields: ['value']
```

Comment:
Do we want to add trailing commas for all lines? You have a trailing comma at line 2191, so just pointing it out for consistency.

Reply:
Ha, good point. I thought to myself "I should start getting into the habit of adding trailing commas. But maybe I should rather start doing it with new drivers than adding it to older drivers to be consistent. Adding trailing commas for all lines is probably more trouble than it's worth - I'll remove the trailing comma.